src/main/java/com/yanolja_final/global/exception/ErrorCode.java

```diff
@@ -7,6 +7,10 @@
 @Getter
 public enum ErrorCode {
 
+    //EMAIL
```

Comment:
```suggestion
    // EMAIL
````
구글 코딩컨벤션 상 주석 후 띄어쓰기 필요한 것으로 알고 있습니다!